backend/src/pong/game/game.spec.ts

```diff
@@ -31,4 +32,61 @@ describe('Game unit tests', () => {
 		expect(service).toBeDefined();
 		expect(controller).toBeDefined();
 	});
+
+	describe('CreateGameDto', () => {
+		const validator = new ValidationPipe({
+			whitelist: true,
+			forbidNonWhitelisted: true,
+			transform: true,
```

Comment:
change to use `globalValidationPipeOptions` after pulling updated main (after merging #51)